app/src/main/java/com/example/medicine/Model/SickLeave.java

```diff
@@ -0,0 +1,71 @@
+package com.example.medicine.Model;
+
+import java.io.Serializable;
+import java.util.ArrayList;
+import java.util.List;
+
+public class SickLeave implements Serializable {
+
+    private String diagnosis;
+    private String startSickLeave;
+    private String endSickLeave;
+    private Integer amountRecipes;
+    private Boolean goToWork;
+    private List<Recipe> recipes;
+
+    public SickLeave init()
+    {
```

Comment:
Скобки должны быть в египетском стиле ([3. Egyptian Brackets](https://blog.codinghorror.com/new-programming-jargon/))
```suggestion
    public SickLeave init() {
```